packages/wonder-blocks-core/util/types.ts

```diff
@@ -119,72 +121,68 @@ export type AriaProps = {|
     "aria-orientation"?: "horizontal" | "vertical",
     "aria-owns"?: IdRefList,
     "aria-posinset"?: number, // integer
-    "aria-pressed"?: "false" | "mixed" | "true" | "undefined",
+    "aria-pressed"?: "false" | "mixed" | "true" | boolean,
     "aria-readonly"?: "false" | "true",
-    "aria-relevant"?: void, // Not using aria-relevant is a best practice
+    "aria-relevant"?: "text" | "all" | "additions" | "additions text" | "removals", // Not using aria-relevant is a best practice
```

Comment:
This was `void` before to prevent people from using this prop.  We could instead use https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/forbid-component-props.md to forbid the use of this prop.